web/src/util/oktaAuth.js

```diff
@@ -0,0 +1,19 @@
+import OktaAuth from '@okta/okta-auth-js';
+
+const OKTA_DOMAIN = process.env.OKTA_DOMAIN; // eslint-disable-line prefer-destructuring
+const OKTA_SERVER_ID = process.env.OKTA_SERVER_ID; // eslint-disable-line prefer-destructuring
+const OKTA_CLIENT_ID = process.env.OKTA_CLIENT_ID; // eslint-disable-line prefer-destructuring
```

Comment:
You could do...

```
const {
  OKTA_DOMAIN,
  OKTA_SERVER_ID,
  OKTA_CLIENT_ID
} = process.env
```

to avoid the `eslint-disable-line` comment.

Reply:
For some reason, I actually couldn't do that, when I tried to do it that
way it seemed like it didn't actually load the values. ¯\_(ツ)_/¯